pb_app/lib/modals.dart

```diff
@@ -32,12 +33,29 @@ class _LoginModalState extends State<LoginModal> {
   // TODO: should this be a workflow?
   Future<void> submitEmailAndPassword() async {
     // TODO: show global loading indicator
-    print('submit!');
 
     try {
-      await client.users
-          .authViaEmail(emailController.text, passwordController.text);
-      // TODO: use success toast to say we succeeded
+      await Workflows.authenticate(
+          email: emailController.text, password: passwordController.text);
+      ScaffoldMessenger.of(context).showSnackBar(
+        SnackBar(
+          backgroundColor: Colors.green,
+          content: Row(
+            children: const [
+              Icon(
+                Icons.check_circle_outline_rounded,
+                color: Colors.white,
+              ),
+              SizedBox(
+                width: 16,
+              ),
+              Text('Signed in successfully!'),
+            ],
+          ),
+        ),
+      );
```

Comment:
I don't think we need to notify the user of success, the navigation to a feature that requires authentication will inform them of the success :)
```suggestion
```